### PR TITLE
Always append locale string to landing page titles

### PR DESCRIPTION
**Pull request by irwalker**
Relates to Vimaly [ticket](https://vimaly.com/#j8mqm/t/www_Landing_page_title_fields_breaking_when_differ.../S6IJ4nHLJPE3iwF92)

This PR changes landing page titles to always append the localised "| Sharesight" string. After it is deployed, the titles for landing pages will need to be manually edited or we will have double ups of e.g. "| Sharesight | Sharesight UK"

### CODE REVIEW

**Comment by irwalker** [comment]
@kylorhall do we also need to make this change for partner pages?

File: extensions/landing-pages.rb [hunk 1]
```diff
@@ -21,22 +22,24 @@ def after_configuration
...
           )
         end
       end
 
-      def process_collection(lang)
-        collection = app.data['landing-pages'].pages
-        collection = collection.map{ |tuple| tuple[1] } # contentful passes ["id", { ... }]
-        collection = collection.map{ |model| localize_entry(model, lang, default_locale_obj[:lang]) }
-        return collection
+      def process_collection(locale_obj)
```
Comment (kylorhall) [inline extensions/landing-pages.rb:36]: Only real question is could we use this `MiddlemanLandingPagesHelpers::landing_pages_collection` (given we use that in another place).

**Comment by kylorhall** [comment]
@irwalker And once you're done with any sort of refactor or just want to have at it, merge it into the `develop` branch and it will auto-deploy for you.  You can then navigate to a landing page on `staging-www.sharesight.com`.

File: spec/helpers/middleman/landing_pages_spec.rb [hunk 2]
```diff
@@ -5,33 +5,57 @@
     @app = Capybara.app
   end
 
-  context "landing_page_url" do
+  describe "landing_page_url" do
```
[the remark this answers is not in the document]
Comment (irwalker) [reply spec/helpers/middleman/landing_pages_spec.rb:8]: I might spend some tinker time on Friday to clean up some of the tests to make them more 'rspecy': https://lmws.net/describe-vs-context-in-rspec

In general I think its cleaner to use `describe` to wrap a piece of functionality and then `context` to wrap a series of tests in a certain situation for that functionality (if that makes sense)

**Comment by irwalker** [comment]
This is deployed to staging: can see that landing pages have the locale appended to the title: https://staging-www.sharesight.com/nz/captainfi/

This will require manual edits of the page titles once deployed to production

**Comment by irwalker** [comment]
Latest commit resolves the bug in the tests that we saw on staging

**Pull request merged**